test-cases/upgrades/kubernetes-scylla-upgrade.yaml

```diff
@@ -0,0 +1,36 @@
+# TODO: need to qualify on GCE and AWS
+
+test_duration: 300
+
+# workloads
+stress_cmd_r: cassandra-stress read no-warmup cl=QUORUM n=2010020 -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=100 -pop seq=1..2010020 -log interval=5
+stress_cmd_w: cassandra-stress write no-warmup cl=QUORUM n=2010020 -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=100 -pop seq=1..2010020 -log interval=5
+
+gce_instance_type_minikube: 'n1-highmem-16'
+gce_root_disk_size_minikube: 240
+k8s_scylla_cpu_n: 2
+k8s_scylla_mem_gi: 8
+k8s_scylla_disk_gi: 30
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+
+user_prefix: 'k8s-scylla-upgrade'
+
+#server_encrypt: true
```

Comment:
Explain why the configuration is commented out

Reply:
fixed